sys/targets/targets.go

```diff
@@ -33,6 +33,7 @@ type Target struct {
 	KernelLinker     string // override LD when running kernel make
 	KernelArch       string
 	KernelHeaderArch string
+	ModuleLoadOffset int
```

Comment:
This must not be here. Target describes static information about an OS, not a particular VM specifics. In some contexts we use completely different kernels/machine types for the same Target.